test-junit-engine-tests/maven/pom.xml

```diff
@@ -0,0 +1,123 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>dev.zio</groupId>
+    <version>1.0</version>
+    <artifactId>zio_test_junit_test</artifactId>
+    <packaging>pom</packaging>
+    <name>${project.artifactId}</name>
+    <description>Testing ZIO Test Junit runner test project</description>
+    <inceptionYear>2020</inceptionYear>
+
+    <properties>
+        <encoding>UTF-8</encoding>
+        <scala.version>2.13.13</scala.version>
+        <scala.compat.version>2.12</scala.compat.version>
+        <zio.version>2.0.22</zio.version>
```

Comment:
What's the reason for using version `2.0.22`?

Reply:
It doesn't matter, it's overwritten by the test with the current version. This is only a template. The concept was taken from "test-junit-tests". The idea is the same as this project: we run simple tests with maven to check that the engine picks them and reports them correctly